File: check_db.js
const { initializeApp, cert } = require('firebase-admin/app');
const { getFirestore } = require('firebase-admin/firestore');

const serviceAccount = require('./serviceAccountKey.json');

initializeApp({
  credential: cert(serviceAccount)
});

const db = getFirestore();

async function check() {
  const q = db.collection('daily_records').orderBy('date', 'desc').limit(1);
  const snap = await q.get();
  snap.forEach(doc => {
    console.log("DATE:", doc.id);
    const data = doc.data();
    console.log("Malam tickets_3a_by_channel:", JSON.stringify(data.malam?.tickets_3a_by_channel, null, 2));
    console.log("Siang tickets_3a_by_channel_visit:", JSON.stringify(data.siang?.tickets_3a_by_channel_visit, null, 2));
  });
}
check();

File: check_db.py
import firebase_admin
from firebase_admin import credentials
from firebase_admin import firestore
import json

cred = credentials.Certificate('serviceAccountKey.json')
firebase_admin.initialize_app(cred)
db = firestore.client()

docs = db.collection('daily_records').order_by('date', direction=firestore.Query.DESCENDING).limit(1).stream()
for doc in docs:
    data = doc.to_dict()
    print("DATE:", doc.id)
    print("Malam tickets_3a_by_channel:", json.dumps(data.get('malam', {}).get('tickets_3a_by_channel'), indent=2))
    print("Siang tickets_3a_by_channel_visit:", json.dumps(data.get('siang', {}).get('tickets_3a_by_channel_visit'), indent=2))
